### PR TITLE
Allows tagging based on github_id

### PR DESCRIPTION
**Pull request by krishvoor**
### Description
This PR allows for efficient cloud management by tagging the cluster & resources
using this format `User:${GITHUB_USERNAME}`

Signed-off-by: Krishna Harsha Voora <krvoora@redhat.com>

### CODE REVIEW

**Comment by krishvoor** [comment]
Hi :wave: @mukrishn @smalleni @dry923 ptal

**Comment by dry923** [comment]
@krishvoor not to be a :hankey: but can you add a description to the PR? :-)

File: dags/openshift_nightlies/scripts/install/rosa.sh [hunk 1]
```diff
@@ -252,7 +252,7 @@ install(){
         if [ $AWS_AUTHENTICATION_METHOD == "sts" ] ; then
             INSTALLATION_PARAMS="${INSTALLATION_PARAMS} --sts -m auto --yes"
         fi
-        rosa create cluster --cluster-name ${CLUSTER_NAME} --version "${ROSA_VERSION}" --channel-group=${MANAGED_CHANNEL_GROUP} --multi-az --compute-machine-type ${COMPUTE_WORKERS_TYPE} --compute-nodes ${COMPUTE_WORKERS_NUMBER} --network-type ${NETWORK_TYPE} ${INSTALLATION_PARAMS}
+        rosa create cluster --tags=user:${GITHUB_USERNAME} --cluster-name ${CLUSTER_NAME} --version "${ROSA_VERSION}" --channel-group=${MANAGED_CHANNEL_GROUP} --multi-az --compute-machine-type ${COMPUTE_WORKERS_TYPE} --compute-nodes ${COMPUTE_WORKERS_NUMBER} --network-type ${NETWORK_TYPE} ${INSTALLATION_PARAMS}
```
Comment (dry923) [inline dags/openshift_nightlies/scripts/install/rosa.sh:255]: There is already a "user" tag on many of the instances. Can we use a different tag name instead to ensure we don't squash something that is already in use

Comment (krishvoor) [reply dags/openshift_nightlies/scripts/install/rosa.sh:255]: Sure, how about  `--tags=perf_org:${GITHUB_USERNAME}`?

Comment (dry923) [reply dags/openshift_nightlies/scripts/install/rosa.sh:255]: how about just github_id ? its painfully obvious then what it tracks back too

**Comment by athiruma** [comment]
@krishvoor tag format will be <User:github_username>. If u feel it has some conflicts let's add some suffix to it ie. <User:github_username_aws_user>. So will know the user from this aws user.

**Comment by krishvoor** [comment]
@dry923 had a chat with @athiruma, looks like they need it as `user:${GITHUB_USERNAME}` for ease of tracking.

**Comment by dry923** [comment]
> @dry923 had a chat with @athiruma, looks like they need it as `user:${GITHUB_USERNAME}` for ease of tracking.

ack

**Comment by krishvoor** [comment]
@athiruma here you go.
@dry923 ptal

**Review by smalleni — APPROVED** [review]

**Review by morenod — APPROVED** [review]
LGTM

**Pull request merged**